src/components/propertiesInput/PropertiesInputData.jsx

```diff
@@ -1,170 +1,251 @@
+                  name='noOfGarage'
+                  placeholder="no-of-garage"
+                  className="property-option"
+                  onChange={informationHandler}
+                >
+                  <option value="" disabled selected hidden>
+                    Select no of Garage
+                  </option>
+                  {PropertyFeature.map((feature) => (
+                    <option value={feature}>{feature}</option>
+                  ))}
+                </select>
               </div>
-              <div className="properties-content-wrapper">
-                <div className="property-location">
-                  <div className="property-location-header">
-                    Property Description
-                  </div>
-                  <div className="property-data-input-fields">
-                    <textarea
-                      name="property-description"
-                      id=""
-                      cols="50"
-                      rows="10"
-                      className="property-desc"
-                      placeholder="property description"></textarea>
-                  </div>
+              <div className="property-data-input-field">
+                <select
+                  value={propertiesInfo.noOfStore}
+                  name='noOfStore'
+                  placeholder="no-of-store"
+                  className="property-option"
+                  onChange={informationHandler}
+                >
+                  <option value="" disabled selected hidden>
+                    Select no of stores
+                  </option>
+                  {PropertyFeature.map((feature) => (
+                    <option value={feature}>{feature}</option>
+                  ))}
+                </select>
+              </div>
+            </div>
+            <div className="properties-content-wrapper">
+              <div className="property-location">
+                <div className="property-location-header">Location</div>
+                <label htmlFor="price">State</label>
+                <InputField
+                  type='text'
+                  placeholder='state'
+                  name='state'
+                  value={propertiesInfo.state}
+                  onChange={informationHandler}
+                />
+                <label htmlFor="price">Address</label>
+               <InputField
+                type='text'
+                placeholder='address'
+                name='address'
+                value={propertiesInfo.address}
+                onChange={informationHandler}
+                />
+              <label htmlFor="price">land Area</label>
+               <InputField
+                type='text'
+                placeholder='land area'
+                name='land_area'
+                value={propertiesInfo.landArea}
+                onChange={informationHandler}
+                />
+              <label htmlFor="price">Property Description</label>
```

Comment:
I suggest you map the input field so you can reduce the lines of code here and also stick to DRY principle